eZ/Publish/SPI/Compare/CompareField.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Compare;
+
+use eZ\Publish\SPI\Persistence\ValueObject;
+
+abstract class CompareField extends ValueObject
```

Comment:
Again, verbs, verbs everywhere... :)

But here I'm having a bit of trouble understanding why Field VO/what's the purpose. TL;DR; the description :sweat_smile: .

Reply:
as stated in desc, it holds data for single field (based on data type) inside FieldType and it is used to chose proper engine for comparison.